### DISCUSSION TITLE
What is your task Management System Outside of Zettelkasten work?

### DISCUSSION THREAD

**Discussion by brandish-transfusible** [BryanJenksCommunity/FAQ#143] [Q&A] [answered]
In earlier videos you were using Taskwarrior. Is this still your goto for total task management? Does that integrate currently with your Obsidian work or does the Obsidian work sync into your task management solution somehow?

Would appreciate your thoughts on this relative to the period where you  have made Obsidian more central to your daily operations.

**Comment by tallguyjenks** [comment, marked as answer]
i dont use obsidian for task management or daily operations other than my personal journal, i don't use task warrior at all right now, and primarily only todoist. This may change when i return back to an arch linux system but at the moment its all todoist
